plugin/adsk/scripts/mayaUsdMenu.mel

```diff
@@ -653,7 +653,7 @@ global proc mayaUsdMenu_markingMenuCallback( string $obj )
         string $tmp = getMayaUsdString("kMayaRefMergeEdits");
         string $pushback = `menuItem -label $tmp -insertAfter "" -image "merge_to_USD.png" -command ("mayaUsdMenu_pushBackToUSD " + $pulledObject)`;
         
-        string $tmp = getMayaUsdString("kMayaRefDiscardEdits");
+        $tmp = getMayaUsdString("kMayaRefDiscardEdits");
```

Comment:
Fixed a script warning. The variable is already declared just above on line 653.

Reply:
Thank you, Sean. Jerry fixed it as well.